models/base/manifest/<YOUR PACKAGE NAME>_base_sessions_lifecycle_manifest.sql

```diff
@@ -0,0 +1,49 @@
+{#
+Copyright (c) 2023-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{{
+  config(
+    materialized='incremental',
+    unique_key='session_identifier',
+    upsert_date_key='start_tstamp',
+    sort='start_tstamp',
+    dist='session_identifier',
+    partition_by = snowplow_utils.get_value_by_target_type(bigquery_val={
+      "field": "start_tstamp",
+      "data_type": "timestamp"
+    }, databricks_val='start_tstamp_date'),
+    cluster_by=snowplow_utils.get_value_by_target_type(bigquery_val=["session_identifier"], snowflake_val=["to_date(start_tstamp)"]),
+    full_refresh=<YOUR PACKAGE NAME>.allow_refresh(),
+    tags=["manifest"],
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt')),
+    tblproperties={
+      'delta.autoOptimize.optimizeWrite' : 'true',
+      'delta.autoOptimize.autoCompact' : 'true'
+    },
+    snowplow_optimize = true
+  )
+}}
+
+{% set sessions_lifecycle_manifest_query = snowplow_utils.base_create_snowplow_sessions_lifecycle_manifest(
+    session_identifiers=var('snowplow__session_identifiers'),
+    session_sql=var('snowplow__session_sql', none),
+    session_timestamp=var('snowplow__session_timestamp', 'collector_tstamp'),
+    user_identifiers=var('snowplow__user_identifiers'),
+    user_sql=var('snowplow__user_sql', none),
+    quarantined_sessions='<YOUR PACKAGE NAME>_base_quarantined_sessions',
+    derived_tstamp_partitioned=var('snowplow__derived_tstamp_partitioned', true),
+    days_late_allowed=var('snowplow__days_late_allowed', 3),
+    max_session_days=var('snowplow__max_session_days', 3),
+    app_ids=var('snowplow__app_id', []),
+    snowplow_events_database=var('snowplow__database', target.database) if target.type not in ['databricks', 'spark'] else var('snowplow__databricks_catalog', 'hive_metastore') if target.type in ['databricks'] else var('snowplow__atomic_schema', 'atomic'),
+    snowplow_events_schema=var('snowplow__atomic_schema', 'atomic'),
+    snowplow_events_table=var('snowplow__events_table', 'events'),
+    event_limits_table='<YOUR PACKAGE NAME>_base_new_event_limits',
+    incremental_manifest_table='<YOUR PACKAGE NAME>_incremental_manifest'
+) %}
+
```

Comment:
This is missing the package name argument, but we're also missing it in unified... I _think_ this only means we maybe are updating the sessions manifest table even when there are no new events, but i'm not 100% sure...

Reply:
Added it in ✔️ . Nice spot on it missing in unified too! Yeah as far as I can follow the `is_run_with_new_events` macro will currently always return true because the package name doesn't match up